extension/src/components/wallet_details.rs

```diff
@@ -57,20 +87,50 @@ impl Component for WalletDetails {
 fn render_balances(balance: &BalanceEntry) -> Html {
     let balance_id = format!("data-cy-{}-balance-text-field", balance.ticker.clone());
     let balance_classes = format!("label {}", balance_id);
+    // todo use enums
+    let ticker_icon = match balance.ticker.as_str() {
+        "L-BTC" => html! {<img width="32px" src={"./bitcoin.svg"} />},
+        "USDt" => html! {<img width="32px" src={"./tether.svg"} />},
+        default => html! {<label class="label">{default}</label>},
+    };
+    let value = format!("{:.4}", balance.value);
     html! {
-        <ybc::Level>
-            <ybc::LevelLeft>
-                <ybc::LevelItem>
-                    <label class="label">{balance.ticker.clone()}</label>
-                </ybc::LevelItem>
-                <ybc::LevelItem>
-                    <label class={balance_classes}>{balance.value.clone()}</label>
-                </ybc::LevelItem>
-            </ybc::LevelLeft>
-        </ybc::Level>
+        <>
+            <ybc::Tile ctx=Parent>
```

Comment:
:upside_down_face: I don't think we need fully qualified syntax for all these `ybc` components.